oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/DurationUtils.java

```diff
@@ -187,6 +189,36 @@ public int secondsBetween(Downsampling downsampling, DateTime dateTime) {
         return durations;
     }
 
+    /**
+     * convert duration to second unit
+     * @param step
+     * @param duration
+     * @return
+     */
+    public long toSecond(Step step, long duration) {
```

Comment:
I think this method is not necessary as the duration unit has been removed from the protocol